crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -62,6 +62,7 @@ pub fn generate_cpp_properties(
 pub mod tests {
     use super::*;
```

Comment:
a compiled example of this in qml-features `naming.rs` would be good to check that rust_name / cxx_name work correctly